main/lsp/LSPIndexer.h

```diff
@@ -48,8 +48,11 @@ class LSPIndexer final {
      */
     bool canTakeFastPath(const LSPFileUpdates &edit,
                          const UnorderedMap<core::FileRef, std::shared_ptr<core::File>> &evictedFiles) const;
-    bool canTakeFastPath(const std::vector<std::shared_ptr<core::File>> &changedFiles,
-                         const UnorderedMap<core::FileRef, std::shared_ptr<core::File>> &evictedFiles) const;
+    /**
+     * INVARIANT: `changedFiles` must have hashes computed.
+     */
```

Comment:
Just OOC: This precondition has always been assumed, it just wasn't previously documented, right?

Reply:
Looks like yes, based on https://github.com/sorbet/sorbet/pull/4145/files#r613591907 .

Reply:
Correct.